src/nupic/os/Env.cpp

```diff
@@ -126,6 +126,8 @@ char ** Env::environ_ = nullptr;
 
 #if defined(NTA_OS_DARWIN)
 #include <crt_externs.h>
+#elif defined(NTA_OS_WINDOWS)
+_CRTIMP extern char **_environ;
```

Comment:
That global is depreciated, and there exists replacement functions to grab environment variables. But i'd hope that any rewrite of Env would slip into 0.3.0  https://msdn.microsoft.com/en-us/library/stxk41x1(VS.80).aspx